pyaedt/edb_core/EDB_Data.py

```diff
@@ -314,24 +314,22 @@ def voids(self):
         return voids
 
     @aedt_exception_handler
-    def points_with_arcs(self):
-        """Return a list of points in [x,y] coordinate.
-        For arc, y is infinite and the arc height is represented with x coordinate.
+    def points_raw(self):
+        """Return a list of Edb points.
 
         Returns
         -------
-        list of list
-            list of [x,y] couple of data.
+        list
+            Edb Points.
         """
         points = []
         try:
             my_net_points = list(self.primitive_object.GetPolygonData().Points)
             for i, point in enumerate(my_net_points):
```

Comment:
@maxcapodi78 We don't need the enumerate anymore because ``i`` is not used.
```suggestion
            my_net_points = list(self.primitive_object.GetPolygonData().Points)
            for point in my_net_points:
```

Reply:
@Alberto-DM  you implemented the enumerate in this method. Is there any specific issue why you didin't iterate directly my_net_points or can I apply suggestion from @MaxJPRey